<launch>
  <node pkg="yac_ros" type="calib_mocap_capture_node" name="calib_mocap_capture_node" required="true" output="screen">
    <param name="rosbag_path" value="/tmp/mocap_data.bag" />
    <param name="imshow" value="true" />

    <!-- <!&#45;&#45; Imperial SRL &#45;&#45;> -->
    <!-- <param name="cam0_topic" value="/camera/color/image_raw" /> -->
    <!-- <param name="body0_topic" value="/vicon/f450/pose" /> -->
    <!-- <param name="body0_topic_type" value="geometry_msgs/PoseStamped" /> -->
    <!-- <param name="target0_topic" value="/vicon/AprilGrid/pose" /> -->
    <!-- <param name="target0_topic_type" value="geometry_msgs/PoseStamped" /> -->

    <!-- UCL AML -->
    <param name="cam0_topic" value="/camera/color/image_raw" />
    <param name="body0_topic" value="/ucl_0/vrpn_client/estimated_odometry" />
    <param name="body0_topic_type" value="nav_msgs/Odometry" />
    <param name="target0_topic" value="/aprilgrid/vrpn_client/estimated_odometry" />
    <param name="target0_topic_type" value="nav_msgs/Odometry" />
  </node>
</launch>
